app/modules/organizations/containers/list/index.js

```diff
@@ -0,0 +1,118 @@
+import { Element as PolymerElement } from 'polymer/polymer-element';
+
+import 'iron-flex-layout/iron-flex-layout-classes';
+
+import 'iron-icons/editor-icons';
+import 'iron-icon/iron-icon';
+
+import 'paper-fab/paper-fab';
+
+import ReduxMixin from 'store';
+import 'shared-styles';
+
+import { fetchOrganizations } from '../../actions';
+import { selectOrganizations } from '../../selectors';
+
+
+export class OrganizationsList extends ReduxMixin(PolymerElement) {
+  static get template() {
+    return `
+      <style include="iron-flex">
+        :host {
+          display: block;
+        }
+        .list-item {
+          background: #fff;
+          -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+          height: 56px;
+          padding: 0 16px;
+          table-layout: fixed;
+          -webkit-transition: .1s all;
+          transition: .1s all;
+          width: 100%;
+
+        }
+        .list-item:hover {
+          background-color: #f2f2f2;
+        }
+        .list-item .cell {
+          line-height: 48px;
+          -webkit-box-sizing: border-box;
```

Comment:
can we autoprefix it?
also, `box-sizing: border-box;` should be applied to all elements globally